src/Education/Commands/RuleDescriptionUserControl.xaml.cs

```diff
@@ -19,14 +19,24 @@
  */
 
 using System.Windows.Controls;
+using System.Windows.Navigation;
+using SonarLint.VisualStudio.Core;
 
 namespace SonarLint.VisualStudio.Education.Commands
 {
     public sealed partial class RuleDescriptionUserControl : UserControl
     {
-        internal RuleDescriptionUserControl()
+        private readonly IBrowserService browserService;
+
+        internal RuleDescriptionUserControl(IBrowserService browserService)
         {
+            this.browserService = browserService;
             InitializeComponent();
         }
+
+        public void HandleRequestNavigate(object sender, RequestNavigateEventArgs e)
+        {
+            browserService.Navigate(e.Uri.AbsoluteUri);
```

Comment:
This call is happening on the UI thread, so an unhandled exception will crash VS.

However, none of the existing uses of this service have exception handling, so I guess it's ok not to have it here.

If we did add exception handling, it would make sense to put it in the browser service itself.